src/main/docs/guide/appendix/architecture/compilerArch.adoc

```diff
@@ -1,6 +1,7 @@
 The Micronaut Compiler is a set of extensions to existing language compilers:
 
-* Java - the link:{jdkapi}/java.compiler/javax/annotation/processing/package-summary.html[Java Annotation Processing (APT) API] is used for Java & Kotlin code (a future version of Micronaut will support KSP for Kotlin)
+* Java - the link:{jdkapi}/java.compiler/javax/annotation/processing/package-summary.html[Java Annotation Processing (APT) API] is used for Java  code.
+* Kotlin - the https://kotlinlang.org/docs/ksp-overview.html[KSP (Kotlin Symbol Processing) API]  is used for Kotlin code.
```

Comment:
We support both [KAPT](https://docs.micronaut.io/latest/guide/#kapt) and [KSP](https://docs.micronaut.io/latest/guide/#ksp) For Kotlin.